media/css/firefox/new/trailhead/download.scss

```diff
@@ -43,10 +43,11 @@ $image-path: '/media/protocol/img';
             border: none;
             color: #fff;
             cursor: pointer;
-            display: inline-block;
+            text-decoration: underline;
+            display: ruby;
+            white-space: nowrap;
```

Comment:
This means the button text will never wrap, so on smaller screens such as mobile it now breaks out of the page.

![image](https://user-images.githubusercontent.com/400117/60437906-42be9080-9c07-11e9-8c3d-6151b8eecb72.png)